### PR TITLE
Add /stackoptions API & add more operation functions

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: src/apis/operations.js [hunk 1]
```diff
@@ -49,17 +49,39 @@ operations.noop = () => {
...
+  options.mode = mode
+
+  return {
+    name: 'composition',
+    options
+  }
+}
+
+operations.blur = (sigma, radius) => {
+  const options = {sigma, radius}
```
Comment (pstadler) [inline src/apis/operations.js]: spaces between braces and vars?

File: test/apis/stackoptions.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import test from 'ava'
+import td from 'testdouble'
+
+import * as transport from '../../src/transport'
+const requestStub = td.replace(transport, 'default')
+
+import rka from '../../src'
+
+test('stackoptions.get', t => {
+  const rokka = rka({ apiKey: 'APIKEY' })
```
Comment (pstadler) [inline test/apis/stackoptions.test.js]: Auth is not required here.

File: src/apis/operations.js (hunk 1 above)
Comment (mweibel) [inline src/apis/operations.js]: fixed

File: test/apis/stackoptions.test.js (hunk 2 above)
Comment (mweibel) [inline test/apis/stackoptions.test.js]: fixed

**Pull request merged**